src/_posts/databases/mongodb/2000-01-01-compass.md

```diff
@@ -1,72 +1,146 @@
+    and use it here
+* Proxy/SSH Tunnel
+    * SSH Tunnel/Proxy Method: SSH with Identity File
+    * SSH Hostname/SSH Port/SSH Username: Depends on the region where your
+    database is hosted. You can find all information at the end of the page.
+    * SSH Identity File: Your private SSH key linked to the public SSH key you
+    uploaded on the [SSH Keys page](https://dashboard.scalingo.com/account/keys)
+    of the Scalingo dashboard.
+* Advanced
+    * Read Preference: Default
+
+{% include mongo_db_tunnel.md %}
+
+### Connection via the DB Tunnel of our CLI
+
+You can also use another method, that consist to create an Encrypted Tunnel
+with our CLI. Then in MongoDB Compass to connect into the database with a
+local address which correspond to the DB Tunnel.
+
+Let's open a DB Tunnel with the Scalingo CLI:
+```bash
+$ scalingo -a my-app db-tunnel SCALINGO_MONGO_URL
+Building tunnel to c393a9e3-42fe-4e33-9e6c-8ee815e9af88.my-app-7093.mongo.a.osc-fr1.scalingo-dbs.com:31312
+You can access your database on:
+127.0.0.1:10000
+```
 
-{% include ssh_endpoints.md %}
+Open MongoDB Compass, then click on "New Connection".
+After that click on "Advanced Connection Options" and fill the fields
+accordingly:
+
+* General
+    * Connection String Scheme: mongodb
+    * Host: 127.0.0.1:10000 (Hostname + Port)
+    * Direct Connection: Yes
+* Authentication
+    * Authentication Method: Username / Password
+    * Authentication Mechanism: Default
+* TLS/SSL
+    * SSL/TLS Connection: Default
+* Proxy/SSH Tunnel
+    * SSH Tunnel/Proxy Method: None
+* Advanced
+    * Read Preference: Default
+
+{% include mongo_db_tunnel.md %}
 
 ### Connection with TLS
 
 As a replacement to using an encrypted tunnel to connect to your database,
 you can make it available on the internet from your database dashboard.
-You must first activate
-[Force TLS]({% post_url databases/mongodb/2000-01-01-start %}#force-tls-connections)
-connections.
+You must activate the 
+[Internet Accessibility]({% post_url platform/databases/2000-01-01-access %}#internet-accessibility).
 
-This is done in Compass by selecting "Server Validation" in the "SSL" section
-of the "Connect to Host" form:
+Open MongoDB Compass, then click on "New Connection".
+After that click on "Advanced Connection Options" and fill the fields
+accordingly:
 
-{% assign img_url = "https://cdn.scalingo.com/documentation/compass/ssl_configuration.png" %}
-{% include mdl_img.html %}
+* General
+    * Connection String Scheme: mongodb
+    * Host: c393a9e3-42fe-4e33-9e6c-8ee815e9af88.my-app-7093.mongo.a.osc-fr1.scalingo-dbs.com:31312 (Hostname + Port)
+* Authentication
+    * Authentication Method: Username / Password
+    * Authentication Mechanism: Default
+* TLS/SSL
+    * SSL/TLS Connection: On
+    * Certificate Authority (.pem): Download the right CA certificate below
+    and use it here
+* Proxy/SSH Tunnel
+    * SSH Tunnel/Proxy Method: None
+* Advanced
+    * Read Preference: Default
```

Comment:
Once again, I don't understand the switch from screenshot to text, I find the screenshot clearer 🤔

Reply:
answered above 😬